training/eval.py

```diff
@@ -24,13 +24,36 @@ def clap_similarity(clap_model_name_or_path, texts, audios, device):
 def wer(asr_model_name_or_path, prompts, audios, device, per_device_eval_batch_size, sampling_rate):
     metric = evaluate.load("wer")
     asr_pipeline = pipeline(model=asr_model_name_or_path, device=device)
+
+    return_language = None
+    if isinstance(asr_pipeline.model, WhisperForConditionalGeneration):
```

Comment:
Whisper is the only ASR model in Transformers where we can compute a language prediction and also a transcription. Otherwise, we would need a language identification model and and ASR model.

Let's keep it simple for the first iteration and assume that 95% of users will use Whisper here, meaning we can use the language prediction from the Whisper model.